### PR TITLE
Changes to include dynamodb resources

### PR DESCRIPTION
**Pull request by prakkarp**
Changes to include dynamodb resources and also changes related to removing is_warmtier_enabled variable.
Updated README.md and example file also.

### CODE REVIEW

File: main.tf.json [hunk 1]
```diff
@@ -201,6 +201,59 @@
                 }
             }
         },
+        {
+            "aws_iam_policy_document": {
+                "clumio_dynamodb_policy_document": {
```
Comment (lawrence-clumio) [inline main.tf.json:206]: Just curious why this one is not protected w/ the `is_dynamodb_enabled` variable?

Comment (lawrence-clumio) [reply main.tf.json:206]: Seems like many of the newly added policies are not protected w/ the variable?

File: main.tf.json [hunk 2]
```diff
@@ -1377,16 +1560,23 @@
...
+                    "protect_dynamodb_version": "${var.is_protect_enabled && var.is_dynamodb_enabled ? \"2.0\" : \"\"}",
                     "protect_ebs_version": "${var.is_protect_enabled && var.is_ebs_enabled ? \"20.1\" : \"\"}",
                     "protect_ec2_mssql_version": "${var.is_protect_enabled && var.is_ec2_mssql_enabled ? \"2.1\" : \"\"}",
                     "protect_rds_version": "${var.is_protect_enabled && var.is_rds_enabled ? \"18.1\" : \"\"}",
                     "protect_s3_version": "${var.is_protect_enabled && var.is_s3_enabled ? \"2.1\" : \"\"}",
-                    "protect_warm_tier_dynamodb_version": "${var.is_protect_enabled && var.is_warmtier_enabled && var.is_warmtier_dynamodb_enabled? \"2.1\" : \"\"}",
-                    "protect_warm_tier_version": "${var.is_protect_enabled && var.is_warmtier_enabled ? \"1.1\" : \"\"}",
+                    "protect_warm_tier_dynamodb_version": "${var.is_protect_enabled && var.is_dynamodb_enabled? \"2.1\" : \"\"}",
+                    "protect_warm_tier_version": "${var.is_protect_enabled && var.is_dynamodb_enabled ? \"1.1\" : \"\"}",
```
Comment (lawrence-clumio) [inline main.tf.json:1579]: hmm this actually seems like we should release a new version of the PostProcess resource as well? What does CFT do, it also tracks Cold DynamoDB support under the Warm Tier version?

File: main.tf.json (hunk 1 above)
Comment (prakkarp) [reply main.tf.json:206]: Yes. New resources should be protected with the appropriate flag. Will make the change.

File: main.tf.json (hunk 2 above)
Comment (prakkarp) [reply main.tf.json:1579]: No. For Cold DynamoDB there is a separate entry:
"protect_dynamodb_version": "${var.is_protect_enabled && var.is_dynamodb_enabled ? \"2.0\" : \"\"}",

We are using the is_dynamodb_enabled  flag to determine whether to set the version for the protect_warm_tier_version and protect_warm_tier_dynamodb_version.

No change is required for this on the provider post_process resource.

File: main.tf.json (hunk 1 above)
Comment (prakkarp) [reply main.tf.json:206]: @lawrence-clumio these are data_source statements. We don't protect the data_sourceAll the resources are protected appropriately

Comment (lawrence-clumio) [reply main.tf.json:206]: Not needed per our conversation.

**Review by lawrence-clumio — APPROVED** [review]

**Pull request merged**